### ISSUE TITLE
[Broken] YT playlist title not extracted

### ISSUE THREAD

**Issue by arter97** [blackjack4494/yt-dlc#109]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dlc version **2020.10.31**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '--flat-playlist', '--add-metadata', '--all-subs', '-J', '-F', 'https://www.youtube.com/playlist?list=RDMBStYsiE618&feature=share&playnext=1']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dlc version 2020.10.31
[debug] Git HEAD: 5713f3ac8c06
[debug] Python version 3.8.5 (CPython) - Linux-5.4.71+-x86_64-with-glibc2.29
[debug] Proxy map: {}
```

## Description

Related to #33(fixed in #57), when providing a playlist(which won't download without #57) to yt-dlc, title is NOT extracted correctly.

Reproducing step:
1. Apply #57 
2.
```
python3.8 ./__main__.py --flat-playlist --add-metadata --all-subs -J -F 'https://www.youtube.com/playlist?list=RDMBStYsiE618&feature=share&playnext=1' > tmp.json
```
3.
```
python3.8 -m json.tool tmp.json > pretty.json
```
4. Open pretty.json

Expected result:
```
{
    "_type": "playlist",
    "entries": [
        # Skipped
    ],
    "id": "RDMBStYsiE618",
    "title": "Mix - BLACKPINK - ‘Lovesick Girls’ 1025 SBS Inkigayo : NO.1 OF THE WEEK",
    "extractor": "youtube:playlist",
    "webpage_url": "https://www.youtube.com/playlist?list=RDMBStYsiE618&feature=share&playnext=1",
    "webpage_url_basename": "playlist",
    "extractor_key": "YoutubePlaylist"
}
```

Actual result:
```
{
    "_type": "playlist",
    "entries": [
        # Skipped
    ],
    "id": "RDMBStYsiE618",
    "extractor": "youtube:playlist",
    "webpage_url": "https://www.youtube.com/playlist?list=RDMBStYsiE618&feature=share&playnext=1",
    "webpage_url_basename": "playlist",
    "extractor_key": "YoutubePlaylist"
}
```

Notice that the title is entirely missing.

As #57 fixes the download itself, I believe this should be on a separate commit/issue to fix.
@insaneracist 

Thanks!

**Comment by blackjack4494** [comment]
It's the playlist title that is missing? Then I believe the regex isn't updated anymore. There are multiple ways to get playlist title. But it seems they changed the position slightly again.

**Comment by arter97** [comment]
Yes, it's the playlist title.

It seems like the ones affected from #57 is returning missing title.

**Comment by undaunt** [comment]
Not sure if this is helpful to know (or not) but both playlist and playlist_title on the latest yt-dlc are not returning title - title returns just NA or null while playlist returns playlist_id (the PLxxxxx guid). yt-dl is properly parsing both playlist and playlist_title as the actual playlist title. I know it's been months since the fork, just thought I'd point out my experience from this week.

**Comment by pukkandan** [comment]
It is fixed in #280 but since this fork has been inactive for a while, you might wanna check out [my fork](https://github.com/pukkandan/yt-dlp)
